### DISCUSSION TITLE
Comparison of Certifications with IDS Certification

### DISCUSSION THREAD

**Discussion by ssteinbuss** [International-Data-Spaces-Association/IDS-RAM_4_0#258] [Q&A]
To which other certifications do we want to compare and can we compare the IDS Certification?

Gaia-X? IEC 62443? More?

**Comment by mokamhuber** [comment]
I think, we have at least the following:
1. the certifications we (partially) build on, i.e., the IEC 62443-4-2, ISO 27001 and BSI C5 
-> it could be interesting to see how big the overlap is, what did we take from them and what they do cover additionally that we decided not to use in our IDS certification
* for Components:
    - ISO 62443-4-2
    - Common Criteria
* for Operational Environments
    - ISO 27001
    - BSI C5
2. Certifications and schemes considered in the GAIA-X context, e.g.
  - EUCS Criteria Catalog
  - CISPE Code of Conduct
3. Maybe Domain-specific certifications?
